pages/who-we-are-hiring.md

```diff
@@ -18,6 +18,12 @@ We're actively hiring for the following positions:
 
 {% include openpositions.html %}
 
+### What's the difference bewteen a Front End Designer and Front End Web Developer?”
+
+We expect our Front End Designers to have design as well as HTML, CSS and some JavaScript experience. They should feel comfortable brainstorming and developing design options to meet user needs; developing wireframes or HTML/CSS prototypes to test their design ideas; and quickly iterating on feedback. They should also feel comfortable working as a front end developer on projects, implementing accessible HTML5 and CSS3. JavaScript experience is a plus, but they are not required to be experts.
+
+By contrast, our Front End Web Developers are not expected to have design experience. Instead, they are expected to have much more experience than our Front End Designers with JavaScript, including strict mode compliance, modularization techniques and tools, and frameworks and libraries. They are expected to be fully comfortable writing vanilla JavaScript, working with APIs, and making front end code perform as efficiently as possible.
```

Comment:
> "By contrast, our Front End Web Developers are not expected to have design experience."

I think it may be important to change this to "visual design" versus "design" because we have so many designs, and we should get used to have descriptors for them.